backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/ddl/views/fresh_Attributes.sql

```diff
@@ -7,6 +7,7 @@ SELECT *
 FROM (
          SELECT CASE
                     WHEN a.Value = '1000015' AND av.value = '01'                                          THEN NULL -- ADR & Keine/Leer
+                    WHEN a.Value = '1000015' AND (av.value IS NOT NULL AND av.value != '') THEN coalesce( nullif(trim(printvalue_override), ''), av.name) -- ADR
```

Comment:
why do we have view and function?

Reply:
yes, Tobi, introduced recently a function;
I saw now